yt_dlp/extractor/radiko.py

```diff
@@ -92,9 +109,11 @@ def _extract_formats(self, video_id, station, is_onair, ft, cursor, auth_token,
         formats = []
         found = set()
         for url_tag in m3u8_urls:
-            pcu = url_tag.find('playlist_create_url')
+            pcu = url_tag.find('playlist_create_url').text
+            if is_onair and not pcu.startswith(self._HOSTS_FOR_LIVE) or (not is_onair and (pcu.startswith(self._HOSTS_FOR_TIME_FREE_FFMPEG_UNSUPPORTED) or pcu.startswith(self._HOSTS_FOR_LIVE)))):
+                continue
```

Comment:
Instead of skipping these formats, maybe it's better to just de-prioritize them? (preference=-10). Opinion, @Lesmiscore ?

Reply:
I would do de-prioritize them, but I doubt keeping non-working formats returned is acceptable

Reply:
Since issue is related with ffmpeg, I think deprioritizing is acceptable. Maybe we can add a `format_note`. A third method would be to set preference=-1000. Then, the format will show in infojson, but not in -F.

Go with whichever you feel is better

Reply:
> Since issue is related with ffmpeg, I think deprioritizing is acceptable. Maybe we can add a `format_note` too.

Let's go with it